<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="R-LAPTOP" failures="1" tests="1" name="nopcommerce-demo-sw-5" time="2.987" errors="0" timestamp="2023-12-10T19:58:05 GMT">
  <testcase classname="com.nopcommerce.demo.testsuite.LoginPageTest" name="verifyUserShouldNavigateToLoginPage" time="0.931">
    <failure type="java.lang.AssertionError" message="Login page not displayed expected [Welcome, Please Sign In!] but found [Welcome, Please Sign In]">
      <![CDATA[java.lang.AssertionError: Login page not displayed expected [Welcome, Please Sign In!] but found [Welcome, Please Sign In]
at com.nopcommerce.demo.testsuite.LoginPageTest.verifyUserShouldNavigateToLoginPage(LoginPageTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserShouldNavigateToLoginPage -->
  <testcase classname="com.nopcommerce.demo.testsuite.LoginPageTest" name="verifyThatUserShouldLogInSuccessFullyWithValidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- verifyThatUserShouldLogInSuccessFullyWithValidCredentials -->
  <testcase classname="com.nopcommerce.demo.testsuite.LoginPageTest" name="verifyErrorMessageWithInvalidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- verifyErrorMessageWithInvalidCredentials -->
  <testcase classname="com.nopcommerce.demo.testsuite.LoginPageTest" name="verifyThatUserShouldLogOutSuccessFully" time="0.0">
    <ignored/>
  </testcase> <!-- verifyThatUserShouldLogOutSuccessFully -->
</testsuite> <!-- nopcommerce-demo-sw-5 -->
